### PR TITLE
Add @wesselversluis

### PR DESCRIPTION
**Pull request by wesselversluis**

### CODE REVIEW

File: site/index.css [hunk 1]
```diff
@@ -1,354 +1,29 @@
 html {
```
Comment (wooorm) [inline site/index.css]: Same here, you’re overwriting our files!

File: site/class-2-style/index.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
+html {
```
Comment (wooorm) [inline site/class-2-style/index.css]: @wesselversluis This file should be removed.

File: site/class-2-style/index.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
+var width = 500,
```
Comment (wooorm) [inline site/class-2-style/index.js]: You should rename all the one-letter variables so they make sense!

File: site/index.css [hunk 4]
```diff
@@ -1,354 +0,0 @@
-html {
```
Comment (wooorm) [inline site/index.css]: You’re removing all our CSS!

File: site/index.html [hunk 5]
```diff
@@ -1,55 +0,0 @@
-<!doctype html>
```
Comment (wooorm) [inline site/index.html]: ...and our HTML? 🤷‍♂️

File: wesselversluis/index.css [hunk 6]
```diff
@@ -0,0 +1,29 @@
+html {
```
Comment (wooorm) [inline wesselversluis/index.css]: Wrong directory?

**Comment by wesselversluis** [comment]
Let's do that. I'll remove this branch and try again

**Pull request closed without merging**